openedx/core/djangoapps/user_api/accounts/__init__.py

```diff
@@ -25,3 +26,13 @@
 
 # Indicates the user's preference that all their account information be private.
 PRIVATE_VISIBILITY = 'private'
+
+# Translators: This message is shown to users who attempt to create a new
+# account using an email address associated with an existing account.
+EMAIL_CONFLICT_MSG = u"It looks like {email_address} belongs to an existing account. " \
```

Comment:
(deleted my old comments)

Yep, I had this before, but erased it because I thought to separate out the message itself from when it's being translated. But I think now it's okay to just pass it around already with the translation function called on it.